webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -329,6 +329,27 @@ export class SakaiRubric extends RubricsElement {
       sakaiItemDelete.requestUpdate("rubric", this.rubric);
     }
   }
+
+ openEditWithKeyboard(e){
+    const classType = e.target.className;
```

Comment:
use classlist for this:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList